recipes/lcr_genie/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "lcr_genie" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/neilswainston/LCRGenie/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f84a26c926cb02f343f79b2facf25bec3ac03a0e337f9bd83a47165c8bccfa68
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install -vv .
+  skip: true  # [win or py != 37]
```

Comment:
This is pure Python and all dependencies seem to be available? Why is this skipping Windows and why isn't it just made `noarch: python`?

Reply:
Hi @dopplershift ,
You're right, I don't know why it didn't work before on windows.
Thank you !!